src/oled.c

```diff
@@ -80,21 +111,105 @@ esp_err_t init_oled(u8g2_t* u8g2)
 	u8g2_InitDisplay(u8g2); 
 
 	u8g2_SetPowerSave(u8g2, 0); 
-    u8g2_ClearBuffer(u8g2);
-    display_logo(u8g2);
-    u8g2_SendBuffer(u8g2);
+    
+    display_sra_logo(u8g2);
+    
+	vTaskDelay(100);
+
+	display_walle_logo(u8g2);
+
+	vTaskDelay(100);
 
 	ESP_LOGI(TAG_OLED, "Configured OLED Display");
 
     return ESP_OK;
 }
 
-esp_err_t display_logo(u8g2_t* u8g2)
+esp_err_t display_sra_logo(u8g2_t* u8g2)
 {
+	// Clear the screen
+	u8g2_ClearBuffer(u8g2);
+
+	// Fill the screen
     u8g2_DrawBox(u8g2, 0, 0, 128, 64);
-    u8g2_DrawBitmap(u8g2,27,0,9,36,sra_logo);
 
+	// Draw Wall-E logo on the screen
+    u8g2_DrawBitmap(u8g2,27,0,9,36,sra_logo);
```

Comment:
```suggestion
    u8g2_DrawBitmap(u8g2, 27, 0, 9, 36, sra_logo);
```
Even though a nitpick, please run a linter to cleanup the code formatting in many places